src/TuDa.CIMS.Web/Components/SubmitPopup.razor.cs

```diff
@@ -0,0 +1,305 @@
+                Name = "Reagenzglas",
+                Note = "Glasgefäß",
+                Price = 7.99,
+                Room = new Room() { Id = Guid.Empty, Name = "Raum-Z" },
+                SerialNumber = "01248",
+                Shop = "Chemieladen2"
+            },
+            PricePerItem = 7.99,
+            Id = Guid.Empty
+        },
+        new()
+        {
+            Amount = 1,
+            AssetItem = new Consumable()
+            {
+                Amount = 100,
+                Id = Guid.Empty,
+                ItemNumber = "123456",
+                Manufacturer = "Röhrchenzentrale",
+                Name = "Analysenröhrchen",
+                Note = "genug im Lager",
+                Price = 17.99,
+                Room = new Room() { Id = Guid.Empty, Name = "Raum-120" },
+                SerialNumber = "0124816",
+                Shop = "Rohr-Shop"
+            },
+            PricePerItem = 17.99,
+            Id = Guid.Empty
+        },
+        new()
+        {
+            Amount = 15,
+            AssetItem = new Consumable()
+            {
+                Amount = 100,
+                Id = Guid.Empty,
+                ItemNumber = "123456",
+                Manufacturer = "Glasmacher",
+                Name = "Becherglas",
+                Note = "Glasgefäß",
+                Price = 7.99,
+                Room = new Room() { Id = Guid.Empty, Name = "Raum-A5" },
+                SerialNumber = "012481012",
+                Shop = "Glas-Shop"
+            },
+            PricePerItem = 7.99,
+            Id = Guid.Empty
+        },
+        new()
+        {
+            Amount = 6,
+            AssetItem = new Consumable()
+            {
+                Amount = 100,
+                Id = Guid.Empty,
+                ItemNumber = "12345686775",
+                Manufacturer = "Glasmacher",
+                Name = "Birnenkolben",
+                Note = "keine Birne",
+                Price = 10.99,
+                Room = new Room() { Id = Guid.Empty, Name = "Raum-B101" },
+                SerialNumber = "0124823467",
+                Shop = "Shop8"
+            },
+            PricePerItem = 7.99,
+            Id = Guid.Empty
+        },
+        new()
+        {
+            Amount = 6,
+            AssetItem = new Consumable()
+            {
+                Amount = 100,
+                Id = Guid.Empty,
+                ItemNumber = "12345646775",
+                Manufacturer = "",
+                Name = "Faltenfilter",
+                Note = "nützlich",
+                Price = 7.99,
+                Room = new Room() { Id = Guid.Empty, Name = "Raum-Z" },
+                SerialNumber = "01248",
+                Shop = "Chemieladen2"
+            },
+            PricePerItem = 7.99,
+            Id = Guid.Empty
+        }
+    ];
+/// <summary>
+/// Returns the price unit of a given asset item as a string.
+/// </summary>
+
+    private static string getPriceUnit(AssetItem assetItem) =>
```

Comment:
🌱 Maybe think about returning the hole PriceString. I provided an example in #95

Reply:
What about the amount? How should it be presented in combination with the PriceString, if necessary?

Reply:
Sorry, I meant the `AmountString` or something like that. Anyway, it's about that you can extract the String concatenation to code instead of HTML